Portfolio.html

```diff
@@ -134,6 +134,10 @@ <h2 class="post">Multi-Post Stories</h2>
     </section>
     <section class="contactForm">
       <div class="contactContainer">
+        <h1 class="contact__title">CONTACT US
+        </h1>
```

Comment:
- [ ] Kindly use an `h2` tag here for best semantic practices 💯 Your site should have only one `h1` tag which should be the main heading of your page